.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Context
+
+AI/gen-ai-hub-sdk-js-backlog#ISSUENUMBER.
+
+Please provide a short description of what your change does and why it is needed.
+
+### Feature scope:
+ 
+- [ ] Task1 
+- [ ] Task2 
+  - [ ] SubTask1 
+
+## Definition of Done
+
+- [ ] Functionality scope stated & covered
```

Comment:
```suggestion
```

Reply:
Why do you want to remove this? Is this because you think `Feature scope` and this check box is redundant?

Reply:
I would still like to keep this one as I removed the `Feature scope` checkboxes now.

Reply:
I don't get what this is supposed to be?
I assume the functionality scope is defined in the block above where you put
```
<!-- Please provide a description of what your change does and why it is needed. -->
```

Reply:
Removed this check-box now.

Reply:
FYI When we originally added this in Java this was also sort of meant as a reminder to create / maintain a meaningful description for your PR 😉